erpnext/stock/doctype/material_request/material_request.js

```diff
@@ -21,7 +21,7 @@ frappe.ui.form.on("Material Request Item", {
 	"qty": function(frm, doctype, name) {
 			var d = locals[doctype][name];
 			if (flt(d.qty) < flt(d.min_order_qty)) {
-				alert(__("Warning: Material Requested Qty is less than Minimum Order Qty"));
+				msgprint(__("Warning: Material Requested Qty is less than Minimum Order Qty"));
```

Comment:
Please use `frappe.msgprint` instead of `msgprint`, global methods like these will be deprecated.